tasks/Node/src/modules/task-utils/deployutil.ts

```diff
@@ -0,0 +1,241 @@
+import fs from 'fs';
+import uuidv4 from 'uuid/v4';
+
+import * as tl from 'azure-pipelines-task-lib/task';
+
+import { DevTestLabsModels, DevTestLabsMappers } from "@azure/arm-devtestlabs";
+import { DeploymentsListByResourceGroupResponse, DeploymentOperationsListResponse } from '@azure/arm-resources/esm/models';
+import { ResourceManagementClient } from '@azure/arm-resources';
+
+function addParameterOverrides(parameterOverrides: string, existingParameters: DevTestLabsModels.ArmTemplateParameterProperties[]): DevTestLabsModels.ArmTemplateParameterProperties[] {
+    if (parameterOverrides == null ||
+        parameterOverrides == undefined ||
+        parameterOverrides.length == 0) {
+        return existingParameters;
+    }
+
+    let newParameter: DevTestLabsModels.ArmTemplateParameterProperties = Object.create(DevTestLabsMappers.ArmTemplateParameterProperties);
+
+    var parsedParameterOverrides = parameterOverrides.split(' ');
```

Comment:
IMO we should avoid using var. Should use either let or const.

Reply:
Agreed. Left over. Fixed.